plugins/GSdx/GSSetting.cpp

```diff
@@ -55,7 +55,9 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_ALPHAHACK:
 			return "Different alpha handling. Can work around some shadow problems.";
 		case IDC_OFFSETHACK:
-			return "Might fix some misaligned fog, bloom, or blend effect.";
+			return "Might fix some misaligned fog, bloom, or blend effect.\n"
+				"The preferred option is Normal (Vertex) as it is most likely to resolve misalignment issues.\n"
+				"The special cases are only useful in couple of games like Captain Tsubasa.\n";
```

Comment:
`\n` should be removed from the end of the line. Also `a` should be inserted before `couple`.